app/src/main/res/layout/topic_overview_fragment.xml

```diff
@@ -72,51 +72,58 @@
       <TextView
         android:id="@+id/see_more_text_view"
         android:layout_width="0dp"
```

Comment:
Rather than full width, referencing from mock, it should be `wrap_content`, also, you might need `app:layout_constraintHorizontal_bias="0.0"` also.

Reply:
done.